### PR TITLE
Add eslint infrastructure

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

**Comment by loay** [comment]
@superkhau PTAL

File: package.json [hunk 1]
```diff
@@ -26,6 +26,8 @@
   },
   "devDependencies": {
     "bluebird": "^3.3.3",
+    "eslint": "^2.13.1",
```
Comment (superkhau) [inline package.json]: in the scripts section add:

```
"lint": "eslint .",
...
"posttest": "npm run lint"
```

**Comment by superkhau** [comment]
One comment, otherwise LGTM.

**Comment by loay** [comment]
@rmg I am assuming nvm doesn't have Oracle installed, correct?

**Comment by rmg** [comment]
The `nvm` environment does have the Oracle headers installed. The problem is likely the result of the 4 different `nvm` configurations running in parallel against the same shared database.

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
Closing PR. Another PR was landed to update eslint infrastructure.

**Pull request closed without merging**